src/components/modal-shorten/index.tsx

```diff
@@ -0,0 +1,93 @@
+import Image from 'next/image';
+import { FC } from 'react';
+
+import IUrl from '@/types/url-type';
+
+interface IModalProps {
+  closeModal: () => void;
```

Comment:
```suggestion
  onClickOutside: () => void;
```
đặt tên như này dễ dàng customize hơn: modal nó không quan tâm em truyền hàm gì vào